food-truck-api/src/main/java/food/truck/api/recommendation/semantic_similarity/FoodTruckThread.java

```diff
@@ -0,0 +1,63 @@
+package food.truck.api.recommendation.semantic_similarity;
+
+import food.truck.api.truck.Truck;
+import org.springframework.data.util.Pair;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Provides a secondary thread to compute tag scores for all trucks
+ */
+public class FoodTruckThread extends Thread {
```

Comment:
Can we rename this to TagScoreThread or similar?

Reply:
Sounds good. I'll rename the file.

Reply:
Done